docs/software-platforms/lsfr/creating-vcs.md

```diff
@@ -0,0 +1,131 @@
+# Creating virtual computers with Lightsail for Research
+
+While working on an [analysis module](../../contributing-to-analyses/analysis-modules/index.md), you can develop locally or on a virtual computer with Lightsail for Research (LSfR).
+
+Virtual computers provide access to a set amount of virtual CPUs (vCPUs), memory, and storage through a web browser rather than a physical machine.
+Using a virtual computer allows you to develop and run analyses that require more CPUs and memory than is available on your local machine.
+
+!!! note
+
+    You will need access to AWS to use LSfR.
+    See [getting access to AWS](../../getting-started/accessing-resources/index.md#getting-access-to-aws) to learn how to get set up with AWS.
+
+- The virtual computers provided through LSfR use an Ubuntu operating system.
+- During setup, you will choose your desired configuration, including the amount of virtual CPUs, memory, and storage space.
+    - You also have the option to add [additional storage to your virtual computer](STUB-LINK for volumes).
```

Comment:
```suggestion
    - You also have the option to add [additional storage to your virtual computer](STUB_LINK for volumes).
```

Reply:
I swear I did this but didn't commit it. Thank you.